### DISCUSSION TITLE
[Feature request] Add a command-line option --config-file=

### DISCUSSION THREAD

**Discussion by w-jablonski** [tsujan/FeatherPad#794] [Ideas]
A command-line option `--config-file=` to specify a custom configuration file location would be useful, hopefully for many.

**Comment by tsujan** [comment]
Converting to a discussion....

**Comment by tsujan** [comment]
Theoretically, many new options could be added. The question is, "Is it worth cluttering the code (and GUI)?"

There are two trends: KDE likes to add as many options as possible, making a mess out of its config dialogs/files; GNOME doesn't want its users to have some options, even when they're needed.

I prefer to add options only when really needed, while keeping the code as manageable as possible.

**Comment by w-jablonski** [comment]
So my "0.001 cents" on this topic is stating the obvious: option `--config-file=` allows users to conveniently have multiple profiles of FeatherPad for various scenarios. In my case, it is a workaround around WM's bug.
